modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/view_version_history.jsp

```diff
@@ -0,0 +1,30 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+ViewVersionHistoryDisplayContext viewVersionHistoryDisplayContext = (ViewVersionHistoryDisplayContext)request.getAttribute(ViewVersionHistoryDisplayContext.class.getName());
+%>
+
+<div class="cms-section">
```

Comment:
I read that this will be in Content, File and All section, so class `cms-section` wouldn't be correct. Do we need a class? If not, we can leave the <div> empty